client/src/stylesheets/components/_ToolList.scss

```diff
@@ -0,0 +1,29 @@
+@import '../base/colors';
+
+.tool-list-body {
+  background-color: $mainwhite;
+
+  ul {
+    list-style-type: square;
+    margin-left: 1em;
+    color: $main-blue;
+  }
+  li {
+    text-align: left;
+    font-size: 88%;
+
+    @media (max-width: 992px) {
```

Comment:
what size screen is this 992px? pull this into _global.scss and give it an identifying label

$tablet-phone: 'only screen and (max-width : 992px)';

Reply:
totally agree but beyond the scope of this PR. We should approach this with media mixins that we can apply everywhere. Will required creating/augmenting _breakpoints.scss and utils/_mixins.scss. 

Opened issue link: https://github.com/codefordenver/Circular/issues/326